official/vision/beta/ops/yolo_ops.py

```diff
@@ -250,7 +250,7 @@ def bbox_iou(bboxes1, bboxes2):
     inter_section = tf.maximum(right_down - left_up, 0.0)
     inter_area = inter_section[..., 0] * inter_section[..., 1]
 
-    union_area = bboxes1_area + bboxes2_area - inter_area
+    union_area = bboxes1_area + bboxes2_area - inter_area + 1e-7
```

Comment:
should we put this into constants?

Reply:
It is not for configuring, but part of the logic, so I think it can stay there. Other parts of the repo does it the same.